### ISSUE TITLE
Unable to locate `cargo-sweep` with a self-hosted runner

### ISSUE THREAD

**Issue by nxsaken** [Leafwing-Studios/cargo-cache#36]
Getting this error when using a self-hosted runner (Amazon Linux) with steps running in a archlinux container:

```
Restoring `cargo-sweep` from cache.
Building `cargo-sweep` from scratch. # builds and installs fine
Creating timestamp.
/__w/_actions/BD103/cargo-sweep/v1/dist/main/index.js:10004
                    throw new Error(`Unable to locate executable file: ${tool}. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.`);
                          ^

Error: Unable to locate executable file: /github/home/.cargo/bin/cargo-sweep. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
    at Object.<anonymous> (/__w/_actions/BD103/cargo-sweep/v1/dist/main/index.js:10004:27)
    at Generator.next (<anonymous>)
    at fulfilled (/__w/_actions/BD103/cargo-sweep/v1/dist/main/index.js:9857:58)
```

Please let me know if you need more info.

**Comment by BD103** [comment]
Thanks for the report! I've seen that reproduce in other Github-hosted runners, so it may be another underlying issue.

**Comment by nxsaken** [comment]
Maybe the `cargo-home` input isn't being used for `cargo-sweep`?

**Comment by adrien-bon** [comment]
Hi!
FYI, reproduced this issue with a Github runner, using `ubuntu:latest`.
CI log= https://github.com/adrien-bon/bevy_ecs_tiled/actions/runs/11474767826/job/31931294568?pr=39
Workflow file = https://github.com/adrien-bon/bevy_ecs_tiled/blob/main/.github/workflows/ci.yml

**Comment by BD103** [comment]
I've opened an issue at https://github.com/BD103/cargo-sweep/issues/8 for this. I plan to publish a fix by this weekend, thanks for your patience!

**Comment by BD103** [comment]
[`v2.0.0`](https://github.com/BD103/cargo-sweep/releases/tag/v2.0.0) has been released, and I'm working on updating `cargo-cache` now!

**Comment by BD103** [comment]
@nxsaken and @adrien-bon, could you please verify that this has been fixed in [`v2.5.0`](https://github.com/Leafwing-Studios/cargo-cache/releases/tag/v2.5.0)? If you have `Leafwing-Studios@v2` in your CI, it should automatically pull in the new version.

Thank you!

**Comment by adrien-bon** [comment]
All good for me! Thanks for the quick fix ! :)
